src/components/Dialogs/FileBrowser/FileBrowserDialogComponent.tsx

```diff
@@ -666,10 +668,26 @@ export class FileBrowserDialogComponent extends React.Component {
                 <div className="file-path">
                     {this.pathItems && (
                         <React.Fragment>
-                            <Tooltip2 content={"Refresh current directory"}>
-                                <AnchorButton className="refresh-button" icon="repeat" onClick={() => fileBrowserStore.selectFolder(fileList.directory, true)} minimal={true} />
-                            </Tooltip2>
-                            <Breadcrumbs className="path-breadcrumbs" breadcrumbRenderer={this.renderBreadcrumb} items={this.pathItems} />
+                            <ButtonGroup>
+                                <Tooltip2 content={"Refresh current directory"}>
+                                    <AnchorButton className="refresh-button" icon="repeat" onClick={() => fileBrowserStore.selectFolder(fileList.directory, true)} minimal={true} />
+                                </Tooltip2>
+                                <Tooltip2 content={this.enableEditPath ? null : "Input directory path"}>
```

Comment:
can be changed to `content="Input directory path" disable={this.enableEditPath}` to avoid blueprint warnings
<img width="399" alt="image" src="https://user-images.githubusercontent.com/43841102/230568732-fba8dec5-2429-4903-afb5-b7c8145c0fde.png">